bundle/language_en.properties

```
@@ -1210,7 +1190,7 @@ angal.newbill.selectamedical.txt
 angal.newbill.selectanexam.txt                                                                         = Select an exam:
 angal.newbill.selectanoperation.txt                                                                    = Select an operation:
 angal.newbill.selectapricelist.title                                                                   = Select a PriceList
-angal.newbill.somepriceshavebeenchangeddoyouwanttoupdatetheitemsprices.fmt.msg                         = Some prices have been changed : {0}\n\nDo you want to update the items' prices?
+angal.newbill.somepriceshavebeenchangeddoyouwanttoupdatetheitemsprices.fmg.msg                         = Some prices have been changed : {0}\n\nDo you want to update the items' prices?
```

```suggestion
angal.newbill.somepriceshavebeenchangeddoyouwanttoupdatetheitemsprices.fmt.msg                         = Some prices have been changed : {0}\n\nDo you want to update the items' prices?
```
and why did you remove some of the standard suffixes like `.txt` in othe rplaces

---

Hi, we're currently looking into this. As far as we are aware, these changes shouldn't have been made, so we are trying to fix them.

---

We've updated the language bundle to contain our changes, so the issue should be fixed!